cola/jax_fns.py

```diff
@@ -131,7 +146,7 @@ def get_default_device():
 def device(device_name):
     del device_name
     zeros = jnp.zeros(1)
-    return zeros.device
+    return zeros.device()
```

Comment:
what if the default device is not the same as device name?

Reply:
@mfinzi you created this function, I'm just making it output the correct thing as in JAX you need to use `array.device()` not as in PyTorch `array.device`